pkg/services/sqlstore/migrations/apikey_mig.go

```diff
@@ -78,4 +78,12 @@ func addApiKeyMigrations(mg *Migrator) {
 		{Name: "key", Type: DB_Varchar, Length: 190, Nullable: false},
 		{Name: "role", Type: DB_NVarchar, Length: 255, Nullable: false},
 	}))
+
+	mg.AddMigration("Add expires to api_key table", NewAddColumnMigration(apiKeyV2, &Column{
+		Name: "expires", Type: DB_BigInt, Nullable: true,
+	}))
+
+	mg.AddMigration("Add index api_key.expires", NewAddIndexMigration(apiKeyV2, &Index{
+		Cols: []string{"expires"}, Type: IndexType,
```

Comment:
Not sure if this index is suitable since you're always filtering by `org_id` when filtering by `expires`. So maybe the index should be `org_id` + `expires`?

Or just skip it altogether given that it shouldn't be that many rows in this table.

What do you think?

Reply:
maybe I remove the index for now and we add it in the future if we experience any performance issues because of this

Reply:
Sounds good :+1: